movies-app/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext, useState } from 'react';
+import { I18N } from '../../core/i18n';
+import { LocaleContext } from '../../core/i18n/locale';
+import './Header.scss';
+import { MovieModal } from '../MovieModal';
+import logo from '../../core/constants/netflix-logo.png';
+import PlusIcon from '../../core/constants/plus-icon.svg';
+
+export const Header = () => {
+  const locale = useContext(LocaleContext);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  return (
+    <>
+      <div className="header">
+        <div className="logo-container">
```

Comment:
Optionally, consider extracting logical groupings of JSX into separate functions insider this components body to improve readability a bit.
E.g `getLogo` and `getSearchBar`.